docs/collect-connect.rst

```diff
@@ -14,6 +14,8 @@ If you'd like to connect Collect to more than one server (or to the same server
 Configure server from QR code
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+ODK uses QR codes for distributing settings and as a form of `passwordless authentication <https://www.cyberark.com/what-is/passwordless-authentication/>`_. Using QR codes to configure authentication between Collect and a server can provide security benefits over traditional passwords but only as long as the QR code is treated securely.
```

Comment:
Let's use https://en.wikipedia.org/wiki/Passwordless_authentication instead. More likely to be up and not have ads or other weird stuff in the future.

Reply:
Much better, thanks!